modules/consensus/synchronize.go

```diff
@@ -433,8 +442,9 @@ func (cs *ConsensusSet) threadedInitialBlockchainDownload() {
 	// Set the deadline 10 minutes in the future. After this deadline, we will say
 	// IBD is done as long as there is at least one outbound peer synced.
 	deadline := time.Now().Add(minIBDWaitTime)
+	numOutboundSynced := 0
 	for {
-		numOutboundSynced := 0
+		numOutboundSynced = 0
```

Comment:
can you explain why you moved `numOutboudSynced` outside the loop? I'm guessing it was a bug?